Commands/Commands/utils.js

```diff
@@ -319,23 +319,27 @@ commands['approve'] = {
       } else {
         checker.getLevel(message.member, (r) => {
           if (state[content[0]].sudo && r !== 2) {
-            message.reply('You need to be an admin to approve this report.').then(deletThis)
-            message.delete()
+            message.reply('You need to be an admin to approve this report.').then(delay(15000)).then((deletThis) => {
```

Comment:
This doesn't work, this way you're passing the function as an argument to the anonymous function to the chain.

```js
.then((msg) => {
  message.delete()
  deletThis(msg)
})
```